notebooks/spark/AerospikeSparkScala.ipynb

```diff
@@ -6,35 +6,81 @@
    "source": [
```

Comment:
The comments from the Python side apply to Scala as well. So when you fix the ones on the Python side, please fix them here as well.